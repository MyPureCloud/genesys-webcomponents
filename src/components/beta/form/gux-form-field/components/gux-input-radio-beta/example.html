<div>
  <gux-input-radio>
    <input
      slot="input"
      type="radio"
      id="dinner-pizza"
      name="dinner"
      value="pizza"
    />
    <label slot="label" for="dinner-pizza">Pizza</label>
  </gux-input-radio>

  <gux-input-radio>
    <input
      slot="input"
      type="radio"
      id="dinner-pasta"
      name="dinner"
      value="pasta"
      checked
    />
    <label slot="label" for="dinner-pasta">Pasta</label>
  </gux-input-radio>

  <gux-input-radio>
    <input
      slot="input"
      type="radio"
      id="dinner-hamburger"
      name="dinner"
      value="hamburger"
    />
    <label slot="label" for="dinner-hamburger">Hamburger</label>
  </gux-input-radio>

  <gux-input-radio>
    <input
      slot="input"
      type="radio"
      id="dinner-sandwich"
      name="dinner"
      value="sandwich"
      disabled
    />
    <label slot="label" for="dinner-sandwich">Sandwich</label>
  </gux-input-radio>
</div>
