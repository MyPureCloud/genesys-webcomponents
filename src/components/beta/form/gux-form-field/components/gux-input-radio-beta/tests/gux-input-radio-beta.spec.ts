import { newSpecPage } from '@stencil/core/testing';
import { GuxInputRadioBeta } from '../gux-input-radio-beta';

describe('gux-input-radio-beta', () => {
  let page: SpecPage;

  beforeEach(async () => {
    page = await newSpecPage({
      components: [GuxInputRadioBeta],
      html: `
      <gux-input-radio-beta>
        <input slot="input" type="radio" id="dinner-sandwich" name="dinner" value="sandwich" disabled>
      </gux-input-radio-beta>
      `,
      language: 'en'
    });
  });

  it('should build', async () => {
    expect(page.rootInstance).toBeInstanceOf(GuxInputRadioBeta);
  });

  it('should render', async () => {
    expect(page.root).toMatchSnapshot();
  });
});
