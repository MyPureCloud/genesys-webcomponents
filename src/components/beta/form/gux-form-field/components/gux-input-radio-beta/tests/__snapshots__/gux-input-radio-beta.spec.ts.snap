// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`gux-input-radio-beta should render 1`] = `
<gux-input-radio-beta>
  <!---->
  <div class="gux-input-radio-container">
    <input disabled="" id="dinner-sandwich" name="dinner" slot="input" type="radio" value="sandwich">
  </div>
</gux-input-radio-beta>
`;
